Applications/MmrMobile/Contracts/AlertEvent.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace HomagConnect.MmrMobile.Contracts
+{
+    public class AlertEvent : IExtensibleDataObject
+    {
+        public AlertEvent(IEnumerable<KeyValuePair<string, string>> localizedSource, IEnumerable<KeyValuePair<string, string>> localizedMessage)
+        {
+            LocalizedSource = localizedSource;
+            LocalizedMessage = localizedMessage;
+        }
+
+        /// <summary>
+        /// Timestamp, when the Event started
+        /// </summary>
+        [JsonProperty(Order = 1)]
+        public DateTimeOffset StartTime { get; set; }
+
+        /// <summary>
+        /// Timestamp for the end of the event
+        /// If the event is still active, the current timestamp is returned
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public DateTimeOffset EndTime { get; set; }
+
+        /// <summary>
+        /// Unique identifier for one event
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public Guid InstanceId { get; set; }
+
+        /// <summary>
+        /// Machine-Number, Format "x-yyy-zz-nnnn"
+        /// </summary>
+        [JsonProperty(Order = 3)]
+        public string? MachineNumber { get; set; }
+
+        /// <summary>
+        /// value = 1 - 1000
+        /// </summary>
+        [JsonProperty(Order = 4)]
+        public short Severity { get; set; }
+
+        /// <summary>
+        /// JSON-String containing the Source of the event in all available languages
+        /// - german and english is mandatory for service
+        /// - optional : customer language
+        /// </summary>
+        [JsonProperty(Order = 5)]
+        public IEnumerable<KeyValuePair<string,string>> LocalizedSource { get; set; }
```

Comment:
Can we use the enumeration https://learn.microsoft.com/en-us/dotnet/api/system.globalization.culturetypes?view=net-8.0#system-globalization-culturetypes-allcultures as key?

Reply:
No. As sometime, the value is just empty.